### PR TITLE
feat: summary bullet points

### PR DESCRIPTION
**Pull request by yili288**
resolves: https://trello.com/c/XArpXw38/17-ui-for-text-summary

### Description
The summary page with two tabs: keyword and points.

### How?
- rearranged the components in Home screen so only the middle content changes between Original page and Summary page.
- used a Tab library from React Native. The component has two parts render tab bar and render scene. Tab bar is the navigation bar at the top and scene is the page itself. Every tab has an unique index and the index state is changed when a tab is selected to control the scene rendered.

### Todo
- get text id for a specific text so it can be used to fetch data such as title, summary etc.
- keywords tab (we don't have plans to do this for capstone but thought I would add it for future use)

https://github.com/yili288/readease/assets/47297703/5d519c3a-b531-46f7-9871-02a24249e0e4

### CODE REVIEW

File: src/screens/SummaryPage.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  const [index, setIndex] = useState(0);
+  const [routes] = useState([
+    // Keywords tab is disabled
+    // { key: 'keywords', title: 'Keywords' },
+    { key: 'points', title: 'Points' },
+  ]);
+
+  const [summaryBulletPoints, setSummaryBulletPoints] = useState("")
+
+  useEffect(() => {
```
Comment (yili288) [inline src/screens/SummaryPage.tsx:34]: fetch summary data

File: src/screens/SummaryPage.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    color: "#000",
+    fontFamily: 'Manrope',
+    fontSize: 20,
+    fontStyle: 'normal',
+    fontWeight: "700",
+    lineHeight: 30,
+    // overwrite library's default styling
+    textTransform: 'none',
+  },
+  titleText: {
```
Comment (yili288) [inline src/screens/SummaryPage.tsx:108]: text styles copied from Home screen

**Comment by yili288** [comment]
I've left out the video summary from this PR because it can be worked on separately.

**Comment by Peteredwardj** [comment]
LGTM! Thanks for adding the network call 👍

**Review by Peteredwardj — APPROVED** [review]

**Pull request merged**